BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -540,7 +540,9 @@ - (void)processResponse:(NSURLResponse *)response
 
 - (void)finishURLSessionTaskWithData:(NSData *)data response:(NSURLResponse *)response error:(NSError *)error
 {
-    [self processResponse:response];
+    if (self.HTTPResponse == nil) {
```

Comment:
could we add a comment why we need to have this check? I assume around duplicated processing of the response?